app/policies/comment_policy.rb

```diff
@@ -1,11 +1,11 @@
 class CommentPolicy < ApplicationPolicy
-  DESTROY_COMMENT_ROLES = %w[superadmin board policy_and_abuse support].freeze
-  DESTROY_ADMIN_POST_COMMENT_ROLES = %w[superadmin board board_assistants_team communications elections policy_and_abuse support].freeze
+  DESTROY_COMMENT_ROLES = %w[superadmin board legal policy_and_abuse support].freeze
+  DESTROY_ADMIN_POST_COMMENT_ROLES = %w[superadmin board board_assistants_team communications elections legal policy_and_abuse support].freeze
   FREEZE_TAG_COMMENT_ROLES = %w[superadmin tag_wrangling].freeze
   FREEZE_WORK_COMMENT_ROLES = %w[superadmin policy_and_abuse].freeze
-  HIDE_TAG_COMMENT_ROLES = %w[superadmin tag_wrangling].freeze
-  HIDE_WORK_COMMENT_ROLES = %w[superadmin policy_and_abuse].freeze
-  SPAM_ADMIN_POST_COMMENT_ROLES = %w[superadmin board board_assistants_team communications elections policy_and_abuse support].freeze
+  HIDE_TAG_COMMENT_ROLES = %w[superadmin legal tag_wrangling].freeze
+  HIDE_WORK_COMMENT_ROLES = %w[superadmin legal policy_and_abuse].freeze
+  SPAM_ADMIN_POST_COMMENT_ROLES = %w[superadmin board board_assistants_team communications elections legal policy_and_abuse support].freeze
```

Comment:
This is for approve/reject, so if not spam then this would need to be broken apart further. To me, though, those imply spam as well. Additionally, both support and PAC have this permission, so I don't see a reason to block Legal from marking things as spam/not spam

Reply:
approve/reject seems to be for spam too. Are you maybe thinking of `can_review_comment?` for "approve moderated admin post comments"? That one is independent of role and always allowed.

But I agree that it sound off to allow legal to delete admin post comments, but not spam/ham admin post comments. So this also sounds like @sarken should take a look.

Reply:
Bilka is right that the issue is referring to moderated comments, meaning this: https://github.com/otwcode/otwarchive/blob/cd20f8e66c426944136f4673c35c70c5a6333548/app/policies/comment_policy.rb#L51-L57 
And it's not something Legal explicitly asked for; it just comes as a little side benefit of being an admin (for now -- we'll eventually limit which admins can do that, but BAT/Comms/NPM were still figuring out workflow and purview when we implemented moderated news post comments).

While there's no _harm_ in giving Legal the ability to spam comments, it's not something they're going to use, so let's leave them out of `SPAM_ADMIN_POST_COMMENT_ROLES`.